packages/js/src/executors/node/node.impl.ts

```diff
@@ -84,10 +86,20 @@ export async function* nodeExecutor(
         mainFolder,
```

Comment:
`mainfolder.replace(projectRoot, '')`

Would this work?

Reply:
I changed it to something similar, yeah.